manager/controllers/packages.go

```diff
@@ -48,6 +49,10 @@ func SystemPackagesHandler(c *gin.Context) {
 		LogAndRespError(c, err, "database error")
 		return
 	}
+	if inventory.PackageData == nil {
+		LogAndRespStatusError(c, http.StatusNoContent,
+			errors.New("No package data available"), "Missing package data")
```

Comment:
this is the cause of lint fail:
~~~
test         | manager/controllers/packages.go:54:4: ST1005: error strings should not be capitalized (stylecheck)

test         | 			errors.New("No package data available"), "Missing package data")
~~~